### PR TITLE
fix header only / status only requests for express api

### PR DESCRIPTION
**Pull request by mamaso**
Fixes missing body issues in #978 for the express api by automatically populating a body property

### CODE REVIEW

File: src/WebJobs.Script/Content/Script/http/response.js [hunk 1]
```diff
@@ -4,6 +4,7 @@
 module.exports = (context) => {
     var res = {
         headers: {},
+        body: undefined,
```
Comment (yvele) [inline src/WebJobs.Script/Content/Script/http/response.js]: What's the point? No property === undefined

Comment (mamaso) [inline src/WebJobs.Script/Content/Script/http/response.js]: It's subtle, the property has value undefined but is not 'undefined'.  Therefore when parsing json in C# we can recognize that there is a body property and flow through the correct response handling path.

File: test/WebJobs.Script.Tests/NodeEndToEndTests.cs [hunk 2]
```diff
@@ -96,7 +96,7 @@ public NodeEndToEndTests(TestFixture fixture) : base(fixture)
             {
                 RequestUri = new Uri(string.Format("http://localhost/api/httptriggerbytearray")),
                 Method = HttpMethod.Post,
-                Content = new ByteArrayContent(inputBytes)
+                Content = new ByteArrayContent(inputBytes),
```
Comment (fabiocav) [inline test/WebJobs.Script.Tests/NodeEndToEndTests.cs]: leftover?

File: test/WebJobs.Script.Tests/NodeEndToEndTests.cs [hunk 3]
```diff
@@ -477,6 +477,27 @@ public void ErrorFunction_DoesNotBringDownHost()
         }
 
         [Fact]
+        public async Task HttpTriggerExpressApi_SendStatus()
+        {
+            HttpRequestMessage request = new HttpRequestMessage
+            {
+                RequestUri = new Uri(string.Format("http://localhost/api/httptrigger")),
+                Method = HttpMethod.Get,
```
Comment (fabiocav) [inline test/WebJobs.Script.Tests/NodeEndToEndTests.cs]: nit: trailing comma

**Pull request merged**